targeteditor/src/main/scala/explore/targeteditor/TargetQueries.scala

```diff
@@ -106,10 +107,10 @@ object TargetQueries {
    * Lens used to change name and coordinates of a target
    */
   val targetPropsL =
-    Lens[TargetResult, (String, SiderealTracking, List[Magnitude])](t =>
-      (t.name, TargetResult.tracking.get(t), t.magnitudes)
+    Lens[TargetResult, (NonEmptyString, SiderealTracking, List[Magnitude])](t =>
+      (NonEmptyString.from(t.name).getOrElse("<EMPTY>"), TargetResult.tracking.get(t), t.magnitudes)
```

Comment:
Maybe this should match the `<New String>` used when creating a new target?

Reply:
Good idea. I'll put it in a constant and reference from both places.